log/view_log.py

```diff
@@ -0,0 +1,10 @@
+from django.http import HttpResponse
+
+def index(request):
+    logs = get_logs()
+    return HttpResponse('<br>'.join(logs))
+
+def get_logs():
+    with open('./_site/community.log') as log_file:
```

Comment:
https://deploy-preview-63--coala-community.netlify.com/static/community.log is empty ;-(

Reply:
This is not how you serve a static file,
read https://docs.djangoproject.com/en/2.0/howto/static-files/